src/media/css/reviews.styl

```diff
@@ -18,6 +18,11 @@ select[name=rating] {
         position: absolute;
         right: 0;
         top: 1px;
+
+        +rtl() {
+            position: static;
+            margin: 3px 5px 0 0;
```

Comment:
I like the way these look next to the email but shouldn't they be attached to the left?

```styl
+rtl() {
    left: 20px;
    margin: 0 4px 0 0;
    right: inherit;

}